packages/terra-date-picker/src/react-datepicker/calendar.jsx

```diff
@@ -44,57 +44,204 @@ const isDropdownSelect = (element = {}) => {
 
 export default class Calendar extends React.Component {
   static propTypes = {
+    /**
+     * Prop to change date when a valid date is selected.
+     */
     adjustDateOnChange: PropTypes.bool,
+    /**
+     * Class name to style the date picker.
+     */
     className: PropTypes.string,
+    /**
+     * Components to render within date picker.
+     */
     children: PropTypes.node,
+    /**
+     * Format of the date selected.
+     */
     dateFormat: PropTypes.oneOfType([
       PropTypes.string,
       PropTypes.array
     ]).isRequired,
+    /**
+     * Prop to style individual days on calendar.
+     */
     dayClassName: PropTypes.func,
+    /**
+     * Prop to select a dropdown mode for the year dropdown.
```

Comment:
This description is not accurate. This prop is used to determine whether the year and month dropdowns should be in the scroll or select mode.

Reply:
Resolved in 1b98d9f